StreamVideoTests/StreamVideo.xctestplan

```diff
@@ -25,7 +25,8 @@
   "testTargets" : [
     {
       "skippedTests" : [
-        "CallCRUDTest\/test_muteAllUsers()"
+        "CallCRUDTest\/test_muteAllUsers()",
+        "CallCRUDTest\/test_pinAndUnpinUser()"
```

Comment:
why are we skipping it?

Reply:
not deployed everywhere atm 😁 Slack for more info